.github/workflows/detect-changes.yml

```diff
@@ -78,12 +78,11 @@ jobs:
         run: git fetch --no-tags origin ${{ env.BASE_BRANCH }}
 
       - name: Set up Node.js
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version-file: .nvmrc
-
-      - name: 📥 Monorepo install
-        uses: ./.github/actions/yarn-nm-install
+          cache: yarn
+      - run: yarn install --immutable
```

Comment:
suggestion: Let's give this task a `name` so it appears with something more meaningful in the UI. For example "Install dependencies"

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/ce525fe4-f69e-4298-9373-2ba561fdd039)